deploy/crds/push_v1alpha1_unifiedpushserver_cr_with_backup.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: push.aerogear.org/v1alpha1
+kind: UnifiedPushServer
+metadata:
+  name: example-ups-with-backups
+spec:
+  backups:
+    # Note that backups here is a list, you can specify more than one
+    -
+      # REQUIRED: The name of a backup here will be the name of the
+      # generated CronJob
+      name: ups-daily-at-midnight
+
+      # OPTIONAL: These labels will be added to the labes of the
+      # CronJob (it will have other standard labels too)
+      labels:
+        "monitoring-key": middleware
```

Comment:
The labels are not required.

Reply:
Yes, labels are not required. These are just optional labels that someone can add (yeah, this 'monitoring-key' example one is probably a bad example, again just copied from the keycloak operator).

Reply:
I think is not required at all. We can remove this part.

Reply:
Yes, I think you're right, I'll remove this labels section now.